code/script.js

```diff
@@ -291,54 +282,44 @@ const selectQuestion = () => {
         category
       }
   } else if (category === 'hat') {
-    currentQuestion = {
-      attribute: 'hat',
-      value: true, 
-      category
+      currentQuestion = {
+        attribute: 'hat',
+        value: true, 
+        category
     } 
   } else if (category === 'glasses') {
-    currentQuestion = {
-      attribute: 'glasses',
-      value: true, 
-      category
-    }  
+      currentQuestion = {
+        attribute: 'glasses',
+        value: true, 
+        category
+      }  
   } else if (category === 'other') {
-    currentQuestion = {
-      attribute: 'smoker',
-      value: true, 
-      category 
+      currentQuestion = {
+        attribute: 'smoker',
+        value: true, 
+        category 
     }
   }
     return currentQuestion
-
 };
   
-
 // This function should be invoked when you click on 'Find Out'.
   // Compare the currentQuestion with the secret person.
   // See if we should keep or remove people based on that
   // Then invoke filterCharacters
   
-
 const checkQuestion = () => {
-
   selectQuestion()
   const secretValue = secret[currentQuestion.attribute]
   if (secretValue === currentQuestion.value) {
     filterCharacters(true,currentQuestion.category);
   } else {
     filterCharacters(false,currentQuestion.category);
   }
```

Comment:
Here, for less code, you can use use a Ternary Operator:
`secretValue === currentQuestion.value ? filterCharacters(true) : filterCharacters(false)`